.github/workflows/ci.yml

```diff
@@ -934,42 +934,42 @@ jobs:
     needs:
       # Basic
       - build_all
-      - build_test_all_windows
+      # - build_test_all_windows
       - build_test_all_bazel
       - test_all
       - test_gpu
 
-      # Subsets
-      - build_test_runtime
-      - build_test_runtime_windows
+      # # Subsets
+      # - build_test_runtime
+      # - build_test_runtime_windows
```

Comment:
These jobs will still run, they just won't be checked by this "summary" step. To fully disable them, comment them out above, or add `false &&` to the `if: needs.setup.outputs.should-run == 'true'` lines

Reply:
ack